drivers/hmis_csv_importer/spec/factories/clean_up_data_sources.rb

```diff
@@ -49,6 +49,17 @@
     end
   end
 
+  factory :importer_force_prioritized_placement_status, class: 'GrdaWarehouse::DataSource' do
```

Comment:
Were there some tests we could easily add to confirm this?  If you take the source files you created and drop them into the fixtures directory you should be able to check the when this is enabled all `PrioritizationStatus` cells are `1` after cleanup and not before.